docs/content/preview/yugabyte-platform/manage-deployments/edit-universe.md

```diff
@@ -14,49 +14,61 @@ menu:
 
-- For information on changing configuration flags, refer to [Edit configuration flags](../edit-config-flags/).
+-> For information on changing configuration flags, refer to [Edit configuration flags](../edit-config-flags/).
 
-- For information on changing user tags, refer to [Create and edit instance tags](../instance-tags/).
+-> For information on changing user tags, refer to [Create and edit instance tags](../instance-tags/).
 
-- For information on changing Kubernetes overrides, refer to [Edit Kubernetes overrides](../edit-helm-overrides/).
+-> For information on changing Kubernetes overrides, refer to [Edit Kubernetes overrides](../edit-helm-overrides/).
 
 ## Edit a universe
 
-To change the configuration of a universe, navigate to **Universes > UniverseName > Actions > Edit Universe**, as per the following illustration:
+To change the configuration of a universe, do the following:
 
-![Edit universe](/images/ee/edit-univ-220.png)
+1. Navigate to your universe and choose **Actions > Edit Universe** to display the **Edit universe** page.
 
-Using the **Edit Universe** page, you can modify the following:
+    ![Edit universe](/images/ee/edit-univ-220.png)
 
-- Regions - you can select any region configured in the provider used to deploy the universe
-- Number of nodes and Availability zones - as you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones
-- [Master placement](../../create-deployments/dedicated-master/)
-- Instance type and volume size - you can select instance types configured in the provider
-- [User tags](../instance-tags/)
+1. Update the configuration.
 
-YugabyteDB Anywhere performs these modifications through the [YB-Masters](../../../architecture/yb-master/) powering the universe. The YB-Masters ensure that the new nodes start hosting the tablet leaders for a set of tablets in such a way that the tablet leader count remains evenly balanced across all the available nodes.
+    Using the **Edit Universe** page, you can modify the following:
 
-Note that you can't change the replication factor of a universe.
+    - **Cloud Configuration**
+        - **Regions** - Select any region configured in the provider used to deploy the universe.
+        - [Master Placement](../../create-deployments/dedicated-master/).
+        - **Total Nodes** and **Availability Zones** - As you add nodes, they are automatically distributed among the availability zones; you can also add, configure, and remove availability zones.
+    - **Instance Configuration** - change instance type and volume size as configured in the provider. In some cases, these operations are available as a [smart resize](#smart-resize).
+    - [User Tags](../instance-tags/). Changing tags doesn't require any node restarts or data migration.
+
+    Note that you can't change the replication factor of a universe.
```

Comment:
May we can include
- **Advanced Configuration** - change communication ports such as masterHttpPort, masterRpcPort, tserverHttpPort, tserverRpcPort. Editing any of these ports causes a Full Move